driver-core/src/main/com/mongodb/client/model/CreateCollectionOptions.java

```diff
@@ -194,6 +199,72 @@ public CreateCollectionOptions collation(@Nullable final Collation collation) {
         return this;
     }
 
+    /**
+     * Returns the expire-after option.  The default value is 0, which indicates no expiration.
+     *
+     * @param timeUnit the time unit
+     * @return the expire-after option, which may be null.
+     * @since 4.3
+     * @mongodb.driver.manual core/timeseries-collections/ Time-series collections
+     */
+    @Nullable
+    public long getExpireAfter(final TimeUnit timeUnit) {
```

Comment:
- Commenting so that GitHub allows me to mark the PR as "reviewed, requested changes".
- I cannot remove this comment because then GitHub removes my the "reviewed, requested changes" status. So I will resolve this comment instead. Sorry for the mess, this is so weird.